src/main/resources/reference.conf

```diff
@@ -172,6 +173,7 @@ resourceTypes = {
           controlled-application-shared-workspace-resource = ["writer", "reader"]
           google-project = ["pet-creator"]
           wds-instance = ["writer"]
+          kubernetes-app = ["manager"]
```

Comment:
I don't think workspace writers or readers should inherit any permissions to `kubernetes-app`. Only workspace owners should be able to manage kubernetes-apps in the workspace.

So remove this line, and the below.